lib/rubocop/cop/style/empty_case_condition.rb

```diff
@@ -80,6 +83,18 @@ def correct_when_conditions(corrector, when_nodes)
             corrector.replace(range, conditions.map(&:source).join(' || '))
           end
         end
+
+        def exist_return_node?(node)
```

Comment:
This entire method can be replaced by:

```ruby
node.each_descendant.any?(&:return_type?)
```

and inlined. 🙂 

🙂

Reply:
Sure! I did not notice this. It will be a simple code 😃 Thanks for your advise.